static/app/views/dashboards/manage/index.tsx

```diff
@@ -71,17 +90,62 @@ function ManageDashboards() {
         query: {
           ...pick(location.query, ['cursor', 'query']),
           sort: getActiveSort().value,
-          per_page: '9',
+          per_page: rows * columns,
         },
       },
     ],
     {staleTime: 0}
   );
 
-  const [showTemplates, setShowTemplatesLocal] = useLocalStorageState(
-    SHOW_TEMPLATES_KEY,
-    shouldShowTemplates()
-  );
+  const dashboardsPageLinks = getResponseHeader?.('Link') ?? '';
+
+  useEffect(() => {
+    function setRowsAndColumns(containerWidth: number) {
+      if (containerWidth === 0) {
+        return;
+      }
```

Comment:
Is `0` special? I'm wondering when this condition happens.

Reply:
I observed this one happening on initial page load. When that happened the number of rows and columns would amount to 0 causing an error in the API call which is the main reason I have that there.

Reply:
nvm just realized i had this check for setting `gridWidth` initially, it's not needed I'll take it out